src/mrack/outputs/pytest_multihost.py

```diff
@@ -70,16 +70,19 @@ def create_multihost_config(self):  # pylint: disable=too-many-branches
         ]
 
         # Use values from provisioning-config mhcfg section - as default values
-        for option in self._config["mhcfg"]:
-            # options defined in metadata takes precedence over options in
-            # provisioning-config as it is "closer" to user.
-            if option in mhcfg:
-                continue
-            mhcfg[option] = self._config["mhcfg"][option]
+        # options defined in metadata takes precedence over options in
+        # provisioning-config as it is "closer" to user.
+        mhcfg = self._config["mhcfg"] | mhcfg
 
         for domain in mhcfg["domains"]:
             for host in domain["hosts"]:
-                provisioned_host = self._db.hosts.get(host["name"])
```

Comment:
This part and the part in util.py are not needed if we do not change the name of host in mrack db (other comment).

Reply:
understood.